service/src/test/java/uk/gov/mca/beacons/api/export/JsonSerialiserUnitTest.java

```diff
@@ -0,0 +1,178 @@
+
+    ownerAddress.setAddressLine1("10 Via Coco");
+
+    owner.setOwnerName("Coco Maria");
+    owner.setCompanyAgent("Cocositos");
+    owner.setCareOf("Haseeb Iqbal");
+    owner.setAddress(ownerAddress);
+    owner.setTelephoneNumbers("01577836277 / 01179822366");
+    owner.setMobiles("07782146399 / 07344800934 / 07321873457");
+    owner.setEmail("cocomaria@gmail.com");
+
+    List<BeaconExportOwnerDTO> owners = List.of(owner);
+    JSONArray jsonOwnersArray = JsonSerialiser.mapBeaconOwnersToJsonArray(
+      owners
+    );
+    JSONObject firstMappedOwner = (JSONObject) jsonOwnersArray.get(0);
+
+    assertEquals("COCO MARIA", firstMappedOwner.get("owner name"));
+    assertEquals("COCOSITOS", firstMappedOwner.get("company agent"));
+    assertEquals("HASEEB IQBAL", firstMappedOwner.get("care of"));
+    assertEquals("COCOMARIA@GMAIL.COM", firstMappedOwner.get("email"));
+  }
+
+  @Test
+  public void mapBeaconOwnersToJsonArray_shouldSeparateMultiplePhoneNumbersWithASemicolon() {
+    BeaconExportOwnerDTO owner = new BeaconExportOwnerDTO();
+    AddressDTO ownerAddress = new AddressDTO();
+
+    ownerAddress.setAddressLine1("10 Via Coco");
+
+    owner.setOwnerName("Coco Maria");
+    owner.setCompanyAgent("Cocositos");
+    owner.setCareOf("Haseeb Iqbal");
+    owner.setAddress(ownerAddress);
+    owner.setTelephoneNumbers("01577836277 / 01179822366");
+    owner.setMobiles("07782146399 / 07344800934 / 07321873457");
+    owner.setEmail("cocomaria@gmail.com");
+
+    List<BeaconExportOwnerDTO> owners = List.of(owner);
+    JSONArray jsonOwnersArray = JsonSerialiser.mapBeaconOwnersToJsonArray(
+      owners
+    );
+    JSONObject firstMappedOwner = (JSONObject) jsonOwnersArray.get(0);
+
+    assertEquals(
+      "01577836277 ; 01179822366",
+      firstMappedOwner.get("telephone numbers")
+    );
+    assertEquals(
+      "07782146399 ; 07344800934 ; 07321873457",
+      firstMappedOwner.get("mobiles")
+    );
+  }
+
+  @Test
+  public void mapBeaconOwnersToJsonArray_whenTheOwnersInformationIsBlank_shouldLeaveTheOwnersBlank() {
+    BeaconExportOwnerDTO blankOwner = new BeaconExportOwnerDTO();
+    AddressDTO blankOwnerAddress = new AddressDTO();
+    blankOwner.setAddress(blankOwnerAddress);
+
+    List<BeaconExportOwnerDTO> owners = List.of(blankOwner);
+    JSONArray jsonOwnersArray = JsonSerialiser.mapBeaconOwnersToJsonArray(
+      owners
+    );
+
+    JSONObject firstMappedOwner = (JSONObject) jsonOwnersArray.get(0);
+    String stringifiedMappedAddress = firstMappedOwner
+      .get("address")
+      .toString();
+
+    assertEquals("", firstMappedOwner.get("owner name"));
+    assertEquals("", firstMappedOwner.get("company agent"));
+    assertEquals("", firstMappedOwner.get("care of"));
+    assertEquals(
+      "{\"address line 1\":\"\",\"country\":\"\",\"address line 2\":\"\",\"address line 3\":\"\",\"address line 4\":\"\",\"postcode\":\"\",\"county\":\"\",\"town or city\":\"\"}",
```

Comment:
It's always better to test using a json lib rather than string. You cannot guarantee the order for a start.